jsonnet/components/kube-state-metrics.libsonnet

```diff
@@ -27,6 +40,22 @@ function(params)
       },
     },
 
+    clusterRole+: {
+      rules+: [
+        {
+          apiGroups: ['autoscaling.k8s.io'],
+          resources: ['verticalpodautoscalers'],
+          verbs: ['list', 'watch'],
+        },
+        // https://github.com/kubernetes/kube-state-metrics/pull/1851/files#diff-916e6863e1245c673b4e5965c98dc27bafbd72650fdb38ce65ea73ee6304e027R45-R47
```

Comment:
Maybe add a comment what one can find behind this link? Something like `With VPA metrics KSM needs to watch customresourcedefinitions`?

Reply:
ACK, done.